photon-client/src/types/WebsocketDataTypes.ts

```diff
@@ -106,7 +86,7 @@ export interface IncomingWebsocketData {
   updatePipelineResult?: WebsocketPipelineResultUpdate;
   networkInfo?: {
     possibleRios: string[];
-    deviceips: string[];
```

Comment:
`deviceips` is the correct name. I agree this name change should be made, see relevant source [here](https://github.com/PhotonVision/photonvision/blob/ededc4f1302b2be251f15c1d15604e4e056e8d41/photon-core/src/main/java/org/photonvision/common/networking/RoborioFinder.java#L48)

Reply:
Sry yes, didn't see that change.